model/otel/metrics.yaml

```
@@ -205,3 +205,37 @@ groups:
           recommended: when applicable
       - ref: error.type
         examples: ["rejected", "timeout", "500", "java.net.UnknownHostException"]
+
+  - id: metric.otel.sdk.exporter.operation.duration
+    type: metric
+    metric_name: otel.sdk.exporter.operation.duration
+    stability: development
+    brief: "The duration of exporting a batch of telemetry items."
```

I think this is more commonly used
```suggestion
    brief: "The duration of exporting a batch of telemetry records."
```